src/main/java/uk/nhs/prm/e2etests/utility/NhsIdentityGenerator.java

```diff
@@ -32,10 +33,6 @@ public static String randomNemsMessageId() {
     }
 
     public static String randomNemsMessageId(boolean shouldLog) {
-        String nemsMessageId = UUID.randomUUID().toString();
-        if (shouldLog) {
-            System.out.println("Generated Nems Message ID " + nemsMessageId);
-        }
-        return nemsMessageId;
+        return UUID.randomUUID().toString();
```

Comment:
the boolean `shouldLog` is unused now. We should either use it or remove it from arguments

Reply:
Addressed